developer-docs-site/docs/nodes/staking.md

```diff
@@ -0,0 +1,236 @@
+
+<ThemedImage
+  alt="Staking Flow"
+  sources={{
+    light: useBaseUrl('/img/docs/staking-light.svg'),
+    dark: useBaseUrl('/img/docs/staking-dark.svg'),
+  }}
+/>
+
+## Joining the validator set
+
+If you are running a validator node, then you can participate in consensus on the Aptos blockchain in a fully permissionless way by joining the validator set. However, to join a validator set:
+
+1. You must stake your Aptos coins with at least the minimum amount required, and
+2. You must lock up these staked coins for at least the minimum lockup duration required. You cannot withdraw any of your staked amount until your lockup period expires.
+
+When you satisfy the above two minimum requirements, then you can join the validator set at any time, start validating and earn rewards.
+
+:::tip Joining the validator set
+For step-by-step instructions on how to join the validator set, see: [Joining Validator Set](https://aptos.dev/nodes/ait/connect-to-testnet#joining-validator-set).
+:::
+
+### Minimum and maximum stake
+
+You must stake the required minimum amount to join the validator set. Moreover, you can only stake up to the maximum stake amount. 
+
+If at any time after joining the validator set, your current staked amount exceeds the maximum allowed stake (for example as the rewards are added to your staked amount), then your voting power and the rewards will be calculated only using the maximum allowed stake amount, and not your current staked amount. 
+
+### When the staked amount falls below minimum
+
+If after joining the validator set, at the start of an epoch your stake drops below the minimum required amount, then you will be removed from the validator set. 
+
+:::tip
+In the current version of the staking on the Aptos blockchain, there is no possibility of your stake dropping below the required minimum before the lockup period expires. **This will change when Aptos implements slashing, i.e., penalty for malicious validator behavior**.
+:::
+
+### Minimum and maximum lockup period
+
+To join the validator set, you must lockup your stake for the required minimum lockup period (in seconds). You cannot withdraw any staked amount before the expiry of the lockup period. 
+
+Moreover, you can only lockup your stake for the maximum allowed period (in seconds).
+
+However, if your remaining lockup time falls below the required minimum lockup period, then you will **not** be removed from the validator set. 
+
+:::tip Note the difference
+If your staked amount falls below the required minimum you will be removed from the validator set, but if your remaining lockup time falls below the required minimum lockup period, you will not be removed from the validator set.
+:::
+
+### When the lockup period expires
+
+When your lockup period expires, you can either extend the lockup period so you can continue to validate and receive the rewards, or you can withdraw your total staked amount and stop validating.
```

Comment:
I think there's a third possibility is they do nothing when lockup period expires, the node will still be validating, but not receiving any reward.